### DISCUSSION TITLE
ダークモード時のチラつきに対する改善要望

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#541] [要望・アイデア・不具合]
ご担当者様

 ダークモード時にタグのページを表示した場合、添付のようにユーザーランキングに表示されるユーザーがチラつきます。
また添付には表示されていませんが、画面中央の記事に表示されるユーザーもチラつきます。

見苦しいため、改善していただけると幸いです。

以上です。よろしくお願いします。

<img width="1010" alt="スクリーンショット 2023-09-27 22 48 48" src="https://github.com/increments/qiita-discussions/assets/38848514/e619c903-9242-4bee-9d2b-1b652ddc27a9">

**Comment by wataru86** [comment]
@Brutus03

ご意見ありがとうございます🙏

> ダークモード時にタグのページを表示した場合、添付のようにユーザーランキングに表示されるユーザーがチラつきます。

ユーザーランキングはパフォーマンスの都合上、後からランキングデータ読み込むようになっており、その間に添付いただいた画像のようにダミーアイテムを表示しています。

現在ダミーアイテムがダークモード時に目立つ配色になっているため、修正を予定しております。しばらくお待ちいただけると幸いです🙇

> また添付には表示されていませんが、画面中央の記事に表示されるユーザーもチラつきます。

こちらはユーザーアイコン画像のことでしょうか？画像の読み込みが完了するまでアイコンが表示されないため、ちらついて見える可能性があります。ご容赦いただけますと幸いです🙇
